src/Http/Adapter/HttpClient.php

```diff
@@ -34,6 +34,19 @@ public function execute(HttpRequestInterface $request): HttpResponseInterface
             $options['proxy'] = self::$proxy;
         }
 
+        $options['verify_peer'] = false;
```

Comment:
I think we should mark this as TODO - to avoid leaving it - this might be security vulnerability in future.

Reply:
removed